Sources/CubeStyleGuide/Values/View+Shadow.swift

```diff
@@ -0,0 +1,28 @@
+//
+//  View+Shadow.swift
+//
+//
+//  Created by Ayodimeji Adejuwon on 05/06/2024.
+//
+
+import SwiftUI
+import CubeFoundationSwiftUI
+
+extension View {
+    func multipleShadows(_ shadows: [FigmaShadow]) -> some View {
```

Comment:
Fab - I added BatchShadow to swim, so you may want to change the function I wrote to use this one :)